src/main/java/uk/ac/cam/cl/dtg/isaac/api/AssignmentFacade.java

```diff
@@ -551,7 +559,8 @@ public Response getAssignmentProgressDownloadCSV(@Context final HttpServletReque
             }
 
             rows.add(totalsRow.toArray(new String[0]));
-            rows.add("Last Name,First Name".split(","));
+            String userInfoHeader = includeUserIDs ? "Last Name, First Name, User ID" : "Last Name,First Name";
```

Comment:
Remove spaces or add whitespace truncation to the line below.

Reply:
Good spot!